app/src/main/java/org/mozilla/reference/browser/browser/BrowserFragment.kt

```diff
@@ -161,6 +163,17 @@ class BrowserFragment : Fragment(), BackHandler, UserInteractionHandler {
                 findInPageBar as FindInPageView),
             owner = this,
             view = view)
+
+        sitePermissionFeature.set(
+            feature = SitePermissionsFeature(
+                anchorView = awesomeBar,
```

Comment:
Shouldn't that be anchored to the toolbar? :)

Reply:
For some reason, when I anchored to the toolbar it doesn't show show up, I guess it has to do with the fact that the toolbar is at the bottom.

Maybe it's showing up, but below the toolbar (outside of our sight) as we didn't expesified in which direction it should be anchored to maybe is take below the a element as a default.

Reply:
Ah, thanks. I'll file an issue for that and investigate.

Let's merge this as-is then. :)

Reply:
👍